MultiLinePlotter/css/styles.css

```diff
@@ -0,0 +1,249 @@
+body {
+    overflow-x: hidden;
+}
+
+.navbar-custom {
+    padding: 12px;
+    background-color: #F1F1F1;
+}
+
+.loklak-brand {
+    font-family: "Courier New";
+    font-size: 35px;
+    text-decoration: none;
+    line-height: 20px;
+    color: rgb(51,51,51);
+    line-height: 20px;
+}
+
+.loklak-brand:hover {
+    text-decoration: none;
+    color: rgb(51,51,51);
+}
+
+.loklak-header {
+    color: #C8254D;
+}
+
+.main {
+    margin-left: 35px;
+    margin-right: 35px;
+}
+
+.input-and-stat {
+
+}
+
+.stat {
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+}
+
+.input {
+    height: 230px;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+}
+
+.input-body {
+    height: 190px;
+}
+
+.plot {
+    height: 300px;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+}
+
+.plot-data {
+  overflow: auto;
+  height: 270px;
+}
+
+.tweet-input {
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    justify-content: space-around;
+    margin-left:10px;
+    margin-right: 10px;
+}
+
+.tweet-search {
+    color: #FFFFFF;
+    background-color: #D81B60;
+}
+
+.tweet-search:hover {
+    color: #FFFFFF;
+}
+
+.tweet-search:focus {
+    color: #FFFFFF;
+}
+
+.tweet-reset {
+    color: #FFFFFF;
+    background-color: #D81B60;
+    margin-left: 10px;
+    display: block;
+    float: right;
+    margin-right: 10px;
+    margin-top: 10px;
+}
+
+.tweet-reset:hover {
+    color: #FFFFFF;
+}
+
+.tweet-reset:focus {
+    color: #FFFFFF;
+}
+
+.tweet-input-field:focus {
+  border-color: #D81B60;
+}
+
+#graph {
+
+}
+
+.error {
+    color: #e57373;
+    border-radius: 4px;
+    border-style: solid;
+    border-width: 0.1px;
+    margin-top: 10px;
+    margin-right: 10px;
```

Comment:
Instead of using `margin-*`. Use just one margin property and specify for all the edges.